apps/microsoft-teams/app-actions/test/mocks.ts

```diff
@@ -285,3 +285,26 @@ export const mockContentType: ContentTypeProps = {
     },
   ],
 };
+
+export const mockTeamInstallation: TeamInstallation = {
```

Comment:
There's also a `mockInstallation` [here](https://github.com/contentful/apps/blob/master/apps/microsoft-teams/app-actions/test/fixtures/mockInstallations.ts). We should probably decide if we want to keep everything in one mock file or have these as separate fixtures.

Reply:
Fixed. I removed the fixture to standardize around the "mocks" mega file.

I guess we're pretty loosy goosey with what's a fixture v.s what's a mock. We should align on that topic :)